go.mod

```diff
@@ -30,8 +30,8 @@ require (
 	github.com/vrischmann/go-metrics-influxdb v0.1.1
 	github.com/xeipuuv/gojsonschema v1.2.0
 	github.com/yudai/gojsondiff v1.0.0
-	golang.org/x/net v0.0.0-20220909164309-bea034e7d591
-	golang.org/x/text v0.3.7
+	golang.org/x/net v0.4.0
+	golang.org/x/text v0.5.0
```

Comment:
Thank you so much @VeronikaSolovei9 for putting out this fix PR! Any reason for pinning the text package version to `v0.5.0`? I see that the latest version for this package is `0.6.0`. Should we/can we move to the latest?

Reply:
Yes, please move to the latest version available.

Reply:
Same for the /net package, the latest version is [v0.5.0](https://pkg.go.dev/golang.org/x/net?tab=versions). Can we please pin it to that?